CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(face_classification)
+
+find_package(catkin REQUIRED COMPONENTS
+	rospy
+  	std_msgs
+	sensor_msgs
+	message_generation
+)
+
+catkin_python_setup()
+
+## Generate messages in the 'msg' folder
+add_message_files(
+	DIRECTORY ros/msg
+	FILES
+	face_classification_single.msg
```

Comment:
Please follow the [ROS naming conventions](http://wiki.ros.org/ROS/Patterns/Conventions#Messages).

FaceClassification + FaceClassificationArray